test/common/common/block_memory_hash_set_test.cc

```diff
@@ -7,7 +7,6 @@
 #include "common/common/block_memory_hash_set.h"
 #include "common/common/fmt.h"
 #include "common/common/hash.h"
-#include "common/stats/stats_options_impl.h"
```

Comment:
I think this file should be fully deleted?

Reply:
Done.